Update OmegaV1GoverningEqns.md

hyungyukang opened the pull request:
This PR slightly modifies the Omega-V1 governing equation document OmegaV1GoverningEqns.md.

- Fix some bugs in formulations
- Update definition and units in `Variable Definitions` to match its usage in the document

Compiled version is [here](https://users.nccs.gov/~hgkang/omega/omega_fix-gov-doc/design/OmegaV1GoverningEqns.html#).

The code needs to be updated (e.g., changing `LayerThickness` to `PseudoThickness`) to match the document. I will open another PR for this.

Checklist
* [x] Documentation:
  * [x] Design document has been generated and [added to the docs](https://github.com/E3SM-Project/Omega/tree/develop/components/omega/doc/design)
  * [x] [Developer's Guide](https://github.com/E3SM-Project/Omega/tree/develop/components/omega/doc/devGuide) has been updated
  * [x] Documentation has been [built locally](https://docs.e3sm.org/Omega/Omega/devGuide/BuildDocs.html) and changes look as expected

hyungyukang committed:
Fix bugs in formulations and improve the document

hyungyukang committed:
Change the location of unit of geopotential

hyungyukang committed:
Move definition of \tilde{W}_tr to notes

hyungyukang commented:
>The code needs to be updated (e.g., changing `LayerThickness` to `PseudoThickness`) to match the document. I will open another PR for this.

I found that this change would require many updates in both Omega and Polaris. I don’t think it’s necessary at this point, so we should keep using `LayerThickness` for $\tilde{h}$ and `GeometricLayerThickness` for $h$.

hyungyukang committed:
Update the name in code for LayerThickness

hyungyukang committed:
Fix bugs in Horizontal Tracer Diffusion

hyungyukang committed:
Remove a note on geometric layer thickness

mark-petersen reviewed (approved):
@hyungyukang thanks for your attention to the documentation. This looks good to me.

hyungyukang committed:
Update definition of unit normal

- Update definition of unit normal since the current definition is
  actually upward normal

hyungyukang committed:
Add unit normal to missing places

xylar commented:
@hyungyukang, I also noticed several issues with this document.  Most are different from your changes but I think a few may overlap.  After we merge your fixes, I would like to rebase #326 and have you take a look at that.

hyungyukang committed:
Update in 'Vertical derivatives in ...'

- Remove \rho_0 in (64) and add \tilde to (63)

xylar commented:
> >The code needs to be updated (e.g., changing `LayerThickness` to `PseudoThickness`) to match the document. I will open another PR for this.

> I found that this change would require many updates in both Omega and Polaris. I don’t think it’s necessary at this point, so we should keep using `LayerThickness` for $\tilde{h}$ and `GeometricLayerThickness` for $h$.

I would be strongly in favor or renaming `LayerThickness` to `PseudoThickness` to avoid confusion in both Omega and Polaris.  I will try to get the ball rolling, even knowing that this will be quite disruptive.  It won't be better if we wait.

xylar commented:
@hyungyukang and @vanroekel, could we get this finalized an merged? I would like to rebase #326 onto that merge and then I have yet another PR (#351) I'd like to make to revise the way the pressure-gradient term is handled.  We ended up needing those revisions for our final implementation.

hyungyukang commented:
@vanroekel, could you please review this PR? I’ve approved #326, and I think we can merge both PRs into develop and open another PR later if needed.

vanroekel reviewed (approved):
Thanks for the ping on this one - approving based on visual inspection

xylar commented:
@vanroekel, can you merge this one? (You're the assignee.) Then, I can rebase and merge #326 later today.

xylar commented:
@vanroekel, it looks like you didn't have time to get to this, which is totally fine.  I'll take care of it now that I'm not on my phone :-).
